tests/scripts/thread-cert/Cert_5_6_05_NetworkDataRegisterAfterAttachRouter.py

```diff
@@ -87,32 +93,12 @@ def test(self):
 
         self.nodes[ROUTER].add_prefix('2001:2:0:1::/64', 'paros')
         self.nodes[ROUTER].add_prefix('2001:2:0:2::/64', 'paro')
+        self.nodes[ROUTER].add_prefix('2001:2:0:3::/64', 'paos')
         self.nodes[ROUTER].register_netdata()
 
         # Set lowpan context of sniffer
         self.simulator.set_lowpan_context(1, '2001:2:0:1::/64')
         self.simulator.set_lowpan_context(2, '2001:2:0:2::/64')
-
-        self.simulator.go(10)
-
-        addrs = self.nodes[ED1].get_addrs()
-        self.assertTrue(any('2001:2:0:1' in addr[0:10] for addr in addrs))
-        self.assertTrue(any('2001:2:0:2' in addr[0:10] for addr in addrs))
-        for addr in addrs:
-            if addr[0:3] == '200':
-                self.assertTrue(self.nodes[LEADER].ping(addr))
-
-        addrs = self.nodes[SED1].get_addrs()
-        self.assertTrue(any('2001:2:0:1' in addr[0:10] for addr in addrs))
-        self.assertFalse(any('2001:2:0:2' in addr[0:10] for addr in addrs))
-        for addr in addrs:
-            if addr[0:3] == '200':
-                self.assertTrue(self.nodes[LEADER].ping(addr))
-
-        self.nodes[ROUTER].add_prefix('2001:2:0:3::/64', 'paos')
-        self.nodes[ROUTER].register_netdata()
-
-        # Set lowpan context of sniffer
```

Comment:
why do we remove these code?

Reply:
@FiveDimensions , is this code removal intentional?

Reply:
align with the test plan